packages/browser/conversion-tests/conversion_test.ts

```diff
@@ -404,7 +404,7 @@ function testGetOrdersResponse(getOrdersResponse: WrapperGetOrdersResponse[]): v
     printer(
         'orderInfo.signedOrder.makerAssetData',
         getOrdersResponse[1].ordersInfos[0].signedOrder.makerAssetData ===
-        '0xf47261b0000000000000000000000000871dd7c2b4b25e1aa18728e9d5f2af4c4e431f5c',
+            '0xf47261b0000000000000000000000000871dd7c2b4b25e1aa18728e9d5f2af4c4e431f5c',
```

Comment:
This was courtesy of my prettier tool. We should probably add a CI step that verifies that the Typescript has been prettified (I'll handle this after the release).